Wrappers/Python/ccpi/processors/CenterOfRotationFinder.py

```diff
@@ -27,30 +27,55 @@ class CenterOfRotationFinder(DataProcessor):
     This processor read in a AcquisitionDataSet and finds the center of rotation 
     based on Nghia Vo's method. https://doi.org/10.1364/OE.22.019078
     
-    Input: AcquisitionDataSet
+    Input: AcquisitionDataSet, slice number to run over
     
     Output: float. center of rotation in pixel coordinate
     '''
     
     def __init__(self):
+
         kwargs = {
-                  
-                  }
-        
+            'slice_number' : 0
+                 }
+
+
         #DataProcessor.__init__(self, **kwargs)
         super(CenterOfRotationFinder, self).__init__(**kwargs)
     
+    #redefining to take slice_number
+    def set_input(self, dataset, slice_number = None):
```

Comment:
as discussed, move `slice_number` definition to appropriate method

Reply:
Changes made. I've added a set_slice() function that must be called after input  data has been set.